docusaurus/docs/create-a-plugin/develop-a-plugin/build-a-data-source-backend-plugin.md

```diff
@@ -175,19 +155,28 @@ Implementing authentication allows your plugin to access protected resources lik
      ...
    }
    ```
-1. Rebuild frontend parts of the plugin to _dist_ directory:
-
-```bash
-yarn build
-```
 
 1. Restart your Grafana instance.
 1. Open Grafana in your web browser.
+1. Verify that alerting is now supported by navigating to your created data source. You should an "Alerting supported" message in the Settings view.
+
+
+### Create an alert
+
+:::note
```

Comment:
It's a good ideea to add spaces before and after note bodies, on account that Prettier may reformat the note in a way that breaks it.